### PR TITLE
fix(pci.project): add missing tracking

### PR DESCRIPTION
**Pull request by vks-ovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/l3-services` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-9892
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~~[ ] Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

## Description

Addition of page tracking missing functionality to handle page track.

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/instances/add/add.routing.js [hunk 1]
```diff
@@ -105,15 +105,35 @@ export default /* @ngInject */ ($stateProvider) => {
...
+        trackPage,
+      ) => (complement, type = 'action', prefix = true) => {
         const name = `${
           prefix ? `${addInstanceTrackPrefix}` : ''
         }${complement}`;
-        return atInternet.trackClick({
-          name,
-          type: 'action',
+        switch (type) {
+          case 'action':
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/project/instances/add/add.routing.js:116]: This case dont have any tracking call ? It should call trackClick

File: packages/manager/modules/pci/src/projects/project/instances/add/add.routing.js [hunk 2]
```diff
@@ -105,15 +105,35 @@ export default /* @ngInject */ ($stateProvider) => {
...
+          case 'navigation':
```
[the remark this answers is not in the document]
Comment (vks-ovh) [reply packages/manager/modules/pci/src/projects/project/instances/add/add.routing.js]: this case I has removed as we are not using it anywhere for now.

File: packages/manager/modules/pci/src/projects/project/instances/add/add.routing.js [hunk 3]
```diff
@@ -105,15 +105,34 @@ export default /* @ngInject */ ($stateProvider) => {
...
-          type: 'action',
+        switch (type) {
+          case 'action':
+            trackClick(name, type);
+            break;
+          case 'page':
+            trackPage(name);
+            break;
+          default:
+            trackClick(name);
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/instances/add/add.routing.js:123]: **nitpick(non-blocking):** 2 cases have same blocks. replace with `if...else`

File: packages/manager/modules/pci/src/projects/project/instances/add/add.routing.js [hunk 4]
```diff
@@ -105,15 +105,25 @@ export default /* @ngInject */ ($stateProvider) => {
...
+        trackClick,
+        trackPage,
+      ) => (complement, type = 'action', prefix = true) => {
         const name = `${
           prefix ? `${addInstanceTrackPrefix}` : ''
         }${complement}`;
-        return atInternet.trackClick({
-          name,
-          type: 'action',
+        return type === 'action' ? trackClick(name, type) : trackPage(name);
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/instances/add/add.routing.js:115]:
```suggestion
        return type === 'page' ? trackPage(name) : trackClick(name, type);
```

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**